package.json

```diff
@@ -38,10 +38,11 @@
         "webpack": "^4.43.0"
     },
     "dependencies": {
-        "chalk": "^4.0.0",
+        "chalk": "^4.1.0",
         "cheerio": "^1.0.0-rc.3",
         "lodash": "^4.17.15",
         "parse5": "^6.0.0",
+        "request": "^2.88.2",
```

Comment:
Request has been recently [deprecated](https://www.npmjs.com/package/request). Perhaps we could use [axios](https://github.com/axios/axios)?